### PR TITLE
Replace class by interface in check documentation

### PR DESCRIPTION
**Pull request by schneidermic0**
Hi, I just found a small typo, I guess.

In the check "Constants Interface Check", the documentation of the exception referred to the class definition instead of the interface definition.

### CODE REVIEW

**Comment by larshp** [comment]
suggest only adding externally accessible links in an externally accessible repository

File: docs/checks/db-access-in-ut.md [hunk 1]
```diff
@@ -60,3 +60,9 @@ The pseudo comment must be placed right after the DB access statement.
 ```ABAP
 SELECT * FROM tadir INTO TABLE @DATA(entries).       "#EC DB_ACCESS_UT
 ```
+
+### Further Readings & Knowledge
+
+* [Clean ABAP - Avoid Usage of TEST-SEAM](https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#use-test-seams-as-temporary-workaround)
```
Comment (eugen-guenther-sap) [inline docs/checks/db-access-in-ut.md:66]: there is no workaround for this check.
you should never access a DB within an Unit Test.

File: docs/checks/db-access-in-ut.md [hunk 2]
```diff
@@ -60,3 +60,9 @@ The pseudo comment must be placed right after the DB access statement.
...
+* [ABAP Test Tools & Frameworks](https://pages.github.tools.sap/EngineeringCulture/ase/ABAP/abapTestTools.html)
```
Comment (eugen-guenther-sap) [inline docs/checks/db-access-in-ut.md:67]: please remove the external link

File: docs/checks/external-call-in-ut.md [hunk 3]
```diff
@@ -22,3 +22,7 @@ In exceptional cases, you can suppress this finding by using the pseudo comment
 ```abap
 SUBMIT program.       “#EC EXT_CALL_UT
 ```
+
+### Further Readings & Knowledge
+
+* [ABAP Test Tools & Frameworks](https://pages.github.tools.sap/EngineeringCulture/ase/ABAP/abapTestTools.html)
```
Comment (eugen-guenther-sap) [inline docs/checks/external-call-in-ut.md:28]: please remove the external link

File: docs/checks/test-seam-usage.md [hunk 4]
```diff
@@ -28,3 +28,5 @@ TEST-SEAM seam_name.    "#EC TEST_SEAM_USAGE
 ### Further Readings & Knowledge
 
 * [Clean ABAP - Avoid Usage of TEST-SEAM](https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#use-test-seams-as-temporary-workaround)
+* [ABAP Test Tools & Frameworks](https://pages.github.tools.sap/EngineeringCulture/ase/ABAP/abapTestTools.html)
```
Comment (eugen-guenther-sap) [inline docs/checks/test-seam-usage.md:31]: please remove the external link

File: docs/checks/unit_test_assert.md [hunk 5]
```diff
@@ -61,5 +61,8 @@ ENDMETHOD.
 ### Further Readings & Knowledge
 
 * [Clean ABAP: Don't obsess about coverage](https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#dont-obsess-about-coverage)
+* [ABAP Test Tools & Frameworks](https://pages.github.tools.sap/EngineeringCulture/ase/ABAP/abapTestTools.html)
```
Comment (eugen-guenther-sap) [inline docs/checks/unit_test_assert.md:64]: please remove the external link

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by schneidermic0** [comment]
I am wondering why 5 other commits are also added to this PR. I only wanted to change this file:
https://github.com/SAP/code-pal-for-abap/blob/1ad05286dddd32572757defcd370ad6e34a3af2b/docs/checks/constants-interface.md

Did this happen, because I have chosen the wrong target branch (`master` instead of `1.1.17`)? If so I could just open a new PR containing only my changes

**Comment by eugen-guenther-sap** [comment]
yes, i saw that you had only one file changed and after i changed the target to 1.17, the other files appeared on the list too.

**Comment by schneidermic0** [comment]
Wouldn't it be easier if I do my changes on `1.1.17` branch directly? I can do this. It's not much effort but we would get a PR for this particular change. Just let me know.

**Comment by eugen-guenther-sap** [comment]
i fixed it.

**Review by ghost — APPROVED** [review]

**Pull request merged**